examples/query-suggestions/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@algolia/query-suggestions",
```

Comment:
For now, our convention is to keep the `-example` suffix.

```suggestion
  "name": "@algolia/query-suggestions-example",
```

Reply:
name modified.